src/services/user.service.js

```diff
@@ -44,6 +63,24 @@ class UserService {
       where: user
     });
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {item} email
+   * @param {item} emailSubject
+   * @param {item} emailBody
+   * @param {item} token
+   * @returns {SendEmailService} @memberof UserService
+   */
+  static sendAccVerificationLink(email) {
```

Comment:
Why is this function in services? does it deal with database? I thought services deals with db only.

Reply:
you can put it may be in controller or...

Reply:
check from eric

Reply:
he was supposed to create a function like this. which you can call whenever you need to send email

Reply:
For this, i think the business logic must not reside in the controller

Reply:
why?

Reply:
Thats how it should be. I have researched on internet.